data.php

```diff
@@ -156,11 +156,12 @@ function readInBlockList() {
         $file="/etc/pihole/gravity.list";
         $linecount = 0;
         $handle = fopen($file, "r");
-        while(!feof($handle)){
+        if(gettype($handle) == "resource") {
```

Comment:
fopen returns false on error, or a resource on success. seems like `if ($handle !== false)` would be better